src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -69,7 +69,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     <AssemblySearchPaths>{CandidateAssemblyFiles};{HintPathFromItem};{TargetFrameworkDirectory};{RawFileName}</AssemblySearchPaths>
     <DesignTimeAssemblySearchPaths>$(AssemblySearchPaths)</DesignTimeAssemblySearchPaths>
   </PropertyGroup>
+  
+  <!-- Default glob settings -->
+  <PropertyGroup>
+    <GlobalExclude>$(GlobalExclude);bin\**;obj\**;packages\**</GlobalExclude>
```

Comment:
I'm matching the Web Sdk here, which put it in the .props.  What's the reason for preferring the .targets?